sycl/test-e2e/Matrix/get_coord_int8_matB_impl.hpp

```diff
@@ -165,6 +182,76 @@ void matrix_sum_cols(big_matrix<T, Rows, Cols> &B,
                                        sum_cols_v.get_host_access());
 }
 
+template <typename T, typename TResult, size_t Rows, size_t Cols,
+          size_t TileRows, size_t TileCols, size_t VNNI>
+void matrix_sum_rows(big_matrix<T, Rows, Cols> &B,
```

Comment:
Minor:
Is there a way to include matrix_sum_rows as part of matrix_sum_cols with may be an extra argument to specify whether we want to sum the rows or cols?

Reply:
Thank you for the suggestion! I have combined `matrix_sum_rows` and `matrix_sum_cols` into a single function called `matrix_sum`.